backend/src/main/java/corea/auth/resolver/AccessedMemberArgumentResolver.java

```diff
@@ -41,6 +41,9 @@ public AuthInfo resolveArgument(MethodParameter parameter, ModelAndViewContainer
         if (accessToken.equals(ANONYMOUS)) {
             return AuthInfo.getAnonymous();
         }
+
+        tokenProvider.validateToken(accessToken);
+
         Long memberId = tokenProvider.getPayload(accessToken).get(ID, Long.class);
 
         return memberRepository.findById(memberId)
```

Comment:
별 말 아니지만 클린코드(?? 확실하지 않음, 아무튼 어느 책에 있어용)에 나온 내용인데,
너무 많은 개행은 오히려 코드 읽는 흐름을 방해해서 다른 개발자가 이해하기 어렵게 만든다 하더라구용~

마지막 두개의 코드는 성격이 비슷해서 붙여도 좋을 것 같아요~
```suggestion

        tokenProvider.validateToken(accessToken);

        Long memberId = tokenProvider.getPayload(accessToken).get(ID, Long.class);
        return memberRepository.findById(memberId)
```